RealEstate/apps/core/static/style.css

```diff
@@ -0,0 +1,122 @@
+/* common */
+
+ul.list-group {
+	padding-top:30px;
+}
+#heading {
+	padding-bottom: 30px;
+}
+.icon-ok {
+	display: none;
+	font-size: 80%;
+	color: green;
+	padding-left: 20px
+}
+.icon-delete {
+    display: none;
+    font-size: 90%;
+    color: #A94431;
+    padding-left: 20px;
+}
+.icon-edit {
+    display: none;
+    color: #31708f;
+    padding-left: 20px;
+    cursor: pointer;
+    font-size: 90%;
+}
```

Comment:
These three icons share mostly the same stuff except for the color. All of them should have "cursor : pointer" as well. (I left that out) Maybe we can make an common class?